lib/ansible/modules/system/aix_filesystem.py

```diff
@@ -0,0 +1,600 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: Kairo Araujo (@kairoaraujo)
+module: aix_filesystem
+short_description: Configure LVM and NFS file systems for AIX.
+description:
+  - This module creates, removes, mount and unmount LVM and NFS file system for
+    AIX using C(/etc/filesystems). For LVM file systems is possible to resize a
+    file system.
+version_added: "2.5"
+options:
+  account_subsystem:
+    description:
+      - Specifies whether the file system is to be processed by the accounting
+        subsystem.
+    choices: ["yes", "no"]
+    default: "no"
+    required: false
+  attributes:
+    description:
+      - Specifies attributes for files system separated by comma.
+    default: agblksize='4096',isnapshot='no'
+    required: false
+  auto_mount:
+    description:
+      - File system is automatically mounted at system restart.
+    choices: ["yes", "no"]
+    default: "yes"
+    required: false
+  device:
+    description:
+      - Logical volume (LV) device name or remote export device to create a NFS
+        file system. It is used to create a file system on an already existing
+        logical volume or the exported NFS file system.
+        If not mentioned a new logical volume name will be created following
```

Comment:
If you want to have this as a new paragraph, you should add `-` at the beginning. The same for the formatting of the description of the `size` option bellow.